File: C#/Tiktok Login Design/UserControl/UserControlSignUpWithEmailCreatAcount.cs
using System.Windows.Forms;

namespace Tiktok_Login_Design
{
    public partial class UserControlSignUpWithEmailCreatAcount
    {
        public UserControlSignUpWithEmailCreatAcount()
        {
            InitializeComponent();
        }
        private void LinkLabel2_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Clear();
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Add(new UserControlSignUpCreatAcount());
        }

        private void LinkLabel3_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Clear();
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Add(new UserControlLogin());
        }
    }
}

File: VB/Tiktok Login Design/UserControl/UserControlSignUpWithEmailCreatAcount.vb
Public Class UserControlSignUpWithEmailCreatAcount
    Private Sub LinkLabel2_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles LinkLabel2.LinkClicked
        Form1.Guna2Panel1.Controls.Clear()
        Form1.Guna2Panel1.Controls.Add(New UserControlSignUpCreatAcount())
    End Sub

    Private Sub LinkLabel3_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles LinkLabel3.LinkClicked
        Form1.Guna2Panel1.Controls.Clear()
        Form1.Guna2Panel1.Controls.Add(New UserControlLogin())
    End Sub
End Class
